### PR TITLE
debugger: pin sqlparse to compatible version

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

**Comment by dcramer** [comment]
This matches what we run for sentry.io, and i confirmed that 2.0 is where they changed the syntax.

File: setup.py [hunk 1]
```diff
@@ -138,6 +138,7 @@
     'statsd>=3.1.0,<3.2.0',
     'structlog==16.1.0',
     'South==1.0.1',
+    'sqlparse>=0.1.16,<0.2.0',
```
Comment (mattrobenolt) [inline setup.py]: Was this a dependency of DDT?

**Review by mattrobenolt — APPROVED** [review]

**Comment by mattrobenolt** [comment]
> and i confirmed that 2.0 is where they changed the syntax.

You mean `0.2.0`. ;)

**Comment by dcramer** [comment]
err yeah 0.2.0

File: setup.py (hunk 1 above)
Comment (dcramer) [inline setup.py]: yeah it is

**Pull request merged**